src/obr/cli.py

```diff
@@ -320,12 +342,17 @@ def status(ctx: click.Context, **kwargs):
         os.chdir(kwargs["folder"])
     project = OpenFOAMProject.get_project()
     filters = kwargs.get("filter")
+    jobs = project.get_jobs(filter=filters or [])
+
+    # check if given path points to valid project
+    if not is_valid_workspace(filters or []):
```

Comment:
Wouldn't this work as well?
```suggestion
    if not is_valid_workspace(filters):
```
Maybe it needs a  `filters = kwargs.get("filter", [])`. Which I think is easier to understand.

Reply:
Ahh I see at line 411 you do it like this
`filters: tuple[str] = kwargs.get("filter", ())`

Reply:
Yes, I am not a big fan of the cli `multiple` option being handled as tuples, either.